tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py

```diff
@@ -0,0 +1,171 @@
+"""Schema for HEADER row of all submission types."""
+
+
+from ...util import SchemaManager
+from ...transforms import calendar_quarter_to_rpt_month_year
+from ...fields import Field, TransformField
+from ...row_schema import RowSchema
+from ... import validators
+from tdpservice.search_indexes.models.tanf import TANF_T6
+
+
+s1 = RowSchema(
+    model=TANF_T6,
+    preparsing_validators=[
+        validators.hasLength(379),
+    ],
+    postparsing_validators=[
```

Comment:
Meta question since I'm doing cat4 stuff now -- are postparsing validators always cat3 validators and vice versa?

Reply:
As it stands all cat3 validators are post parsing validators and vice versa.